### ISSUE TITLE
Fully Supervised Training

### ISSUE THREAD

**Issue by infinite0522** [Elin24/P2RLoss#6]
How are hyperparameters set during fully supervised training?

I tried to train the model for the QNRF dataset with the following hyperparameter settings; the best MAE on the test set is only 94.

DATA:
  BATCH_SIZE: 8
  NUM_WORKERS: 4
  PIN_MEMORY: True

MODEL:
  NAME: 'VGG16BN'
  FACTOR: 1
  LOSS: 'P2R'

TRAIN:
  START_EPOCH: 0
  EPOCHS: 1000
  BASE_LR: 5e-5
  BACKBONE_LR: 1e-5
  WEIGHT_DECAY: 1e-4
  CLIP_GRAD: 5.0
  AUTO_RESUME: False
  LR_SCHEDULER:
    NAME: 'step'
    DECAY_EPOCHS: 20
    DECAY_RATE: 0.95

**Comment by Elin24** [comment]
My setting:
```
DATA:
  BATCH_SIZE: 16
  DATASET: shha
  DATA_PATH: /home/visal/wlin38/crowd/data/QNRF_PNG_SE_1536/
  LABEL_PERCENT: 1.0
  LABEL_PROTOCOL: ../dac_label/ucf-100.txt
  NUM_WORKERS: 8
  PIN_MEMORY: false
EVAL_MODE: false
MODEL:
  FACTOR: 1
  LOSS: P2R
  NAME: VGG16BN
  RESUME: ''
OUTPUT: exp/ucf-L100/output
PRINT_FREQ: 100
SAVE_FREQ: 5
SEED: 2024
TAG: ucf-L100
TEST:
  CROP: true
TRAIN:
  AUTO_RESUME: true
  BACKBONE_LR: 1.0e-05
  BASE_LR: 5.0e-05
  CLIP_GRAD: 5.0
  EPOCHS: 1500
  LR_SCHEDULER:
    DECAY_EPOCHS: 3500
    DECAY_RATE: 0.9
    NAME: step
  START_EPOCH: 0
  USE_CHECKPOINT: false
  WEIGHT_DECAY: 0.0001
```
You can try two fully supervised methods:
1. Write a `ucf-100.txt` as the `LABEL_PROTOCOL: ../dac_label/ucf-100.txt`, using the code to train. In this case, the semi-supervised part works as a balancer against annotation noise.
2. Remove the semi-supervised counting parts, and then train the model with only human-annotated (noised) data. In this case, you should set the crop_size ([here](https://github.com/Elin24/P2RLoss/blob/8e99a561431778e487fd459e435be0d9a2446736/datasets/shha.py#L44)) as 512x512 for better performance.

**Comment by infinite0522** [comment]
Thanks for your reply. I'll try the methods you recommend.
